_pages/about-us/teams/products-and-platforms.md

```diff
@@ -1,10 +1,12 @@
 ---
 title: Products and Platforms
 ---
-Products and Platforms is one of the 5 business units within 18F. We strive to incubate tools for digital government.
```

Comment:
Works for me! One small note: I'd keep _more effectively_ in — without it, the sentence could be read to imply that (without our help), partner agencies aren't delivering on their missions.